notebooks/05_operationalize/pipeline-demo/aml_config/library.yml

```diff
@@ -0,0 +1,21 @@
+environment:

```

Comment:
same comment as above, is this needed or can it be auto-generated?

Reply:
it can be auto-generated. I will delete it in the PR